src/styles/_variables.scss

```diff
@@ -20,6 +20,8 @@ $link-blue: #2e55b9;
 $green2: #06d6a0;
 $aqua: #1E94A3;
 $black-60: #7f7f7f;
+$white: #fff;
+$gray-70: #e5e5e5;
```

Comment:
this color isn't in the design system: https://www.figma.com/file/2t8Wcyu8FAXxzQGpyK8r4f/TC-Design-System-2.0-%2F-2022?node-id=2%3A548

we should only use colors in the design system. plz find the closest color. 

also, please use the color names in the design system. the names in grey are below the color block.

this looks like it's probably closest to black-10.